### PR TITLE
add some infomation in migrating_sse.md from sse to lsx

### PR DESCRIPTION
**Pull request by wangchenguang123**
X86

fe8b72eaed3ebb483e163b49aa90e7e

ad2324f23e79c89771a5db2ebefcc6e

410c42c7e8bfca1ff3f0cc4b9ab7a19

3a0c999a53c44931da3d7f0f29d0a35
LoongArch

ed383c4457607527756aca8537c83f6

22670e8d777031764fa9622fa4005fd

a95fe179dcd69d8e19d1ff40755b2ea

8d9ab195799d3ccddbc63bbb31e3e60

### CODE REVIEW

File: docs/migrating_sse.md [hunk 1]
```diff
@@ -27,8 +27,8 @@ Here is a table of a mapping from SSE intrinsics to their LSX counterpart (WIP):
 | _mm_adds_epi8           | __lsx_vsadd_b                         |
 | _mm_adds_epu16          | __lsx_vsadd_hu                        |
 | _mm_adds_epu8           | __lsx_vsadd_bu                        |
-| _mm_addsub_pd           |                                       |
-| _mm_addsub_ps           |                                       |
+| _mm_addsub_pd           | __lsx_vfmadd_d + __lsx_vxor_v         |
```
Comment (jiegec) [inline docs/migrating_sse.md]: muladdsub is not addsub?

File: docs/migrating_sse.md [hunk 2]
```diff
@@ -41,7 +41,7 @@ Here is a table of a mapping from SSE intrinsics to their LSX counterpart (WIP):
 | _mm_blend_epi16         |                                       |
 | _mm_blend_pd            |                                       |
 | _mm_blend_ps            |                                       |
-| _mm_blendv_epi8         |                                       |
+| _mm_blendv_epi8         | __lsx_vbitsel_v                       |
```
Comment (jiegec) [inline docs/migrating_sse.md]: There is a slight difference: __blendv_epi8 only checks the MSB of each byte, however __lsx_vbitsel_v is bitwise.

File: docs/migrating_sse.md [hunk 3]
```diff
@@ -179,8 +179,8 @@ Here is a table of a mapping from SSE intrinsics to their LSX counterpart (WIP):
 | _mm_cvtsd_si64          |                                       |
 | _mm_cvtsd_si64x         |                                       |
 | _mm_cvtsd_ss            |                                       |
-| _mm_cvtsi128_si32       |                                       |
-| _mm_cvtsi128_si64       |                                       |
+| _mm_cvtsi128_si32       | __lsx_vpickve2gr_{b/bu/h/hu/w/wu}     |
```
Comment (jiegec) [inline docs/migrating_sse.md]: __lsx_vpickve2gr_w

File: docs/migrating_sse.md [hunk 4]
```diff
@@ -179,8 +179,8 @@ Here is a table of a mapping from SSE intrinsics to their LSX counterpart (WIP):
...
+| _mm_cvtsi128_si64       | __lsx_vpickve2gr_{d/du}               |
```
Comment (jiegec) [inline docs/migrating_sse.md]: __lsx_vpickve2gr_d

File: docs/migrating_sse.md (hunk 2 above)
Comment (wangchenguang123) [reply docs/migrating_sse.md]: thank you this is my mistake

File: docs/migrating_sse.md (hunk 1 above)
Comment (wangchenguang123) [reply docs/migrating_sse.md]: 图片
like this it use __lsx_vxor_v to do symbol flipping (a * b) - c

**Comment by wangchenguang123** [comment]
LSX
图片
X86-SSE
图片

**Review by jiegec — APPROVED** [review]

**Pull request merged**